Docs: Byob documentation updated

shyogesh-sw opened the pull request:
Issue #, if available: 

Description of changes:  Updated the BYOB documentation. Please review and approve the changes. Will be requesting your review on the Well architected framework feedback through the same branch in a few hours from now.

Checklist:

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

<!-- For major releases please provide internal ticket id -->

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

shyogesh-sw committed:
docs: byob changes

shyogesh-sw committed:
Merge branch 'feat-bring-your-own-bucket' into byob-docs

SanketD92 commented on docs/docs/user_guide/sidebar/common/studies/data_sources.md:
```diff
@@ -10,64 +10,74 @@ Service Workbench can host Studies internally, and provide access to External St
+Removing external Studies is not supported at this time and it will be available in a future release.
 
-In the meantime, to remove access to the data, in the external study account, delete the CloudFormation stack in the AWS account to remove the permissions enabling access to the data.
+To remove access to the data in the external study account, delete the CloudFormation stack in the AWS account. This removes permissions enabling access to the data.
```
How about this:
To remove access to the data in the external study account, delete the CloudFormation stack in the AWS account **(specified in the "Onboarding an external Study account section above")**. This removes permissions enabling access to the data.

shyogesh-sw replied:
Thanks. Changed as per your feedback

SanketD92 commented on docs/docs/user_guide/sidebar/common/studies/studies_page.md:
```diff
@@ -6,29 +6,25 @@ sidebar_label: Studies Page
 
-### Changing Organization Study Permissions
-To change permissions for an Organization Study:
+### Changing organization's Study permissions
```
"Organization" is a category of studies, similar to "My Studies" and "Open Data". The term "organization's" implies that the study belongs to some organization, which might not be 100% valid.

suggestion: Perhaps change this to say "Changing Organization-categorized Study permissions"..?

shyogesh-sw replied:
Thanks for the feedback. Incorporated the change.

shyogesh-sw committed:
docs:well architected and byob corrections

shyogesh-sw committed:
Documentation updates for well architected review and some BYOB changes

erge branch 'byob-docs' of https://github.com/shyogesh-sw/service-workbench-on-aws into byob-docs

SanketD92 reviewed (approved):
LGTM
